### ISSUE TITLE
Devstack Error: "The async task did not return valid JSON"

### ISSUE THREAD

**Issue by v1k0d3n** [romana/romana#126]
Creating placeholder until I can look deeper and provide more troubleshooting time towards this issue, but I'll put it out there for now.

Running into issues creating a devstack deployment per instructions.

```
TASK [stack/devstack/install-controller : Run devstack "stack" script. This may take around 10 minutes.] ***
fatal: [bj916b-controller]: FAILED! => {"changed": false, "failed": true, "msg": "The async task did not return valid JSON: Extra data: line 2 column 1 - line 14 column 1 (char 122 - 794)"}
```

**Comment by v1k0d3n** [comment]
maybe related to Ansible v2.2.0? Lots of warnings during the run about loop.

```
 [WARNING]: The loop variable 'item' is already in use. You should set the `loop_var` value in the `loop_control` option for the task to something else to avoid variable collisions and unexpected behavior.

 [WARNING]: The loop variable 'item' is already in use. You should set the `loop_var` value in the `loop_control` option for the task to something else to avoid variable collisions and unexpected behavior.
```

**Comment by cgilmour** [comment]
It could be.
Can you provide the full command-line you're running, output of `ansible --version` and `git rev-parse HEAD` in the romana repo you've cloned?
I can try to reproduce it from there.

Those warnings are from a bit of refactoring, will be fixed up at some point.

**Comment by cgilmour** [comment]
I've attempted to reproduce this, and the installation completed as-expected.
If you have another go at getting it working, and get stuck with the same problem (or any other issue), can you update this ticket?
You can also reach us via slack.

**Comment by v1k0d3n** [comment]
let me try again now...sorry, I've just been really slammed.

**Comment by cgilmour** [comment]
There's no rush, we'll be available to help whenever you get back to this.

**Comment by v1k0d3n** [comment]
same issue, same place. i know that development workstations drift a bit from a new environment. this is on a week old OS X Macbook Pro, running El Capitan and I am using `zsh`, so let me know if bash is a requirement (although I did try in `bash` before reaching out).

```
TASK [stack/devstack/install-controller : Run devstack "stack" script. This may take around 10 minutes.] ***
fatal: [bj916b-controller]: FAILED! => {"changed": false, "failed": true, "msg": "The async task did not return valid JSON: Extra data: line 2 column 1 - line 14 column 1 (char 122 - 791)"}

NO MORE HOSTS LEFT *************************************************************

PLAY RECAP *********************************************************************
bj916b-compute01           : ok=32   changed=20   unreachable=0    failed=0
bj916b-controller          : ok=37   changed=22   unreachable=0    failed=1
localhost                  : ok=3    changed=2    unreachable=0    failed=0
$ 
$ ansible --version
ansible 2.2.0
  config file = ~/projects/romana/romana-install/ansible.cfg
  configured module search path = Default w/o overrides
$ git rev-parse HEAD
f88245787c82299c556bb493a13156a99c4da791
```

**Comment by v1k0d3n** [comment]
definitely. sorry for leaving this open for so long @cgilmour
